src/main/java/ch/njol/skript/lang/function/FunctionRegistry.java

```diff
@@ -0,0 +1,599 @@
+package ch.njol.skript.lang.function;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.SkriptAPIException;
+import com.google.common.base.Preconditions;
+import org.jetbrains.annotations.ApiStatus.Internal;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.lang.converter.Converters;
+
+import java.util.*;
+import java.util.Map.Entry;
+import java.util.stream.Collectors;
+
+/**
+ * A registry for functions.
+ *
+ * @author Efnilite
```

Comment:
we don't typically use author tags, though you could add yourself as a CODEOWNER potentially?